Sources/Moya/MoyaProvider+Internal.swift

```diff
@@ -181,19 +181,19 @@ private extension MoyaProvider {
 
     func sendUploadFile(_ target: Target, request: URLRequest, callbackQueue: DispatchQueue?, file: URL, progress: ProgressBlock? = nil, completion: @escaping Completion) -> CancellableToken {
         let uploadRequest = manager.upload(file, with: request)
-        let alamoRequest = target.validate ? uploadRequest.validate() : uploadRequest
+        let alamoRequest = target.validationType == nil ? uploadRequest : uploadRequest.validate(statusCode: target.validationType!.statusCodes)
```

Comment:
nit: [up for discussion, since this actually might be more confusing] can we change these lines to `target.validationType?.statusCodes.map(uploadRequest.validate) ?? uploadRequest` instead?

Reply:
Not sure how this is supposed to work as it cause a build error.
If I'm not mistaken, `map` would call the validate func for every `statusCode`, that's not what we want. Also, we have to specifically call the `validate(statusCode:)` func because there are others `validate` func using another first parameter name.

But yeah, It would be great to find a way to get rid of the force unwrap while keeping the code that simple (though as the nil test is performed before using it it won't cause a crash)